apps/plato_single_variant/Readme.md

```diff
@@ -24,26 +21,26 @@ There are two input options required to run the app:
+  - SNP file: User-specified list of SNPs to include in the analysis. The file format should be either one `RSID` per line or positions in rage format (`chr`,`start`,`stop`,`label`)
 
-### Options
+## Options
 
 ### REGRESSION OPTIONS
 **Regression**:
 PLATO app can perform various kind of regression methods:
--   **Regress-auto** : It is default regression method for the app. It will perform either a linear or logistic regression based on the phenotype provided to the app. It works well when you have quantitative and categorical outcomes in the input phenotype file.
--   **Linear**: This option performs a simple (ordinary least-squares) regression between SNP and outcome/s.
--   **Logistic**: This option performs logistic regression between SNP and outcome/s. The logistic regression is performed using a Newton-Rhapson iteratively reweighted least squares algorithm.
--   **Firth**:  This option perform Firth bias reduction to the models being tested (for details see Firth, 1993 and Heinze and Shempel, 2002). This option can be helpful when using many categorical covariates as control variables. Also, this can be helpful when testing a large number of dichotomous phenotypes with widely varying case/control ratios, as you may see in a PheWAS.
+  - **Regress-auto** : It is default regression method for the app. It will perform either a linear or logistic regression based on the phenotype provided to the app. It works well when you have quantitative and categorical outcomes in the input phenotype file.
+  - **Linear**: This option performs a simple (ordinary least-squares) regression between SNP and outcome/s.
+  - **Logistic**: This option performs logistic regression between SNP and outcome/s. The logistic regression is performed using a Newton-Rhapson iteratively reweighted least squares algorithm.
+  - **Firth**:  This option perform Firth bias reduction to the models being tested (for details see Firth, 1993 and Heinze and Shempel, 2002). This option can be helpful when using many categorical covariates as control variables. Also, this can be helpful when testing a large number of dichotomous phenotypes with widely varying case/control ratios, as you may see in a PheWAS.
 
 **Association Type**:
-- **PheWAS**: Using this option, app will perform association test iteratively on all traits/outcomes provided in the input Phenotype file. It is same as  `--phewas` command in PLATO (See PLATO Manual)
-- **GWAS**: Use this option to perform association test on a single outcome/trait like in GWAS analysis. When this option is used, user need to specify column name of outcome/trait in *Outcome* option of the app.
+  - **PheWAS**: Using this option, app will perform association test iteratively on all traits/outcomes provided in the input Phenotype file. It is same as  `--phewas` command in PLATO (See PLATO Manual)
```

Comment:
- **PheWAS**: Using this option, app will perform association test iteratively on all traits/outcomes provided in the input Phenotype file. It is the same as  `--phewas` command in PLATO (see PLATO manual).